core/unix/signal_private.h

```diff
@@ -280,11 +284,9 @@ typedef struct rt_sigframe {
 #    elif defined(AARCHXX)
     kernel_siginfo_t info;
     kernel_ucontext_t uc;
-    char retcode[RETCODE_SIZE];
```

Comment:
I believe AArch32 has retcode in the frame set up by the kernel.  I think it is not just x86: also AArch32.

Reply:
Yes, you're right, there is a `retcode` in AArch32, but not in AArch64.